docs/Lizzard73_bio.md

```diff
@@ -0,0 +1,7 @@
+#Lizzard73
+
+##Name: Liz
+##Occupation: Time Traveler
+##Location: Oz
```

Comment:
Please specify where in Oz, for example: The Emerald City, Land of the Munchkins, the Wicked Witches Tower, etc...